frame/contracts/src/rent.rs

```diff
@@ -301,8 +302,9 @@ where
 
 	/// Process a report that a contract under the given address should be evicted.
 	///
-	/// Enact the eviction right away if the contract should be evicted and return true.
-	/// Otherwise, **do nothing** and return false.
+	/// Enact the eviction right away if the contract should be evicted and return the amount
+	/// of balance that the contract had while it was alive.
```

Comment:
does it returns this ? I think it returns the `rent_payed + rent_to_pay`.

Reply:
You are right. These docs are wrong.